src/libMessage/ZilliqaMessage.proto

```diff
@@ -755,6 +756,13 @@ message LookupSetMicroBlockFromLookup
     required ByteArray signature         = 3;
 }
 
+message LookupForwardTxnsFromSeed
+{
+    optional ProtoTransactionArray shardtransactions = 1;
+    optional ProtoTransactionArray dstransactions    = 2;
+}
```

Comment:
Message need to add pubkey and signature to verify lookup authenticity

Reply:
message is from seed not lookup. https://github.com/Zilliqa/Issues/issues/425

Reply:
Will this issue be fixed in this PR or seperately?